src/python/pants/backend/graph_info/tasks/list_owners.py

```diff
@@ -51,3 +52,7 @@ def console_output(self, targets):
       if owner_info.values():
         for address_spec in owner_info.values()[0]:
           yield address_spec
+
+  @deprecated("1.8.0.dev3", "Run './pants --owner-of=<file> list' instead")
```

Comment:
Sorry, the docs aren't clear here... based on https://www.pantsbuild.org/deprecation_policy.html , let's target this to `1.10.0.dev0`.